Content.Client/Input/ContentContexts.cs

```diff
@@ -86,6 +86,12 @@ public static void SetupContexts(IInputContextContainer contexts)
             human.AddFunction(ContentKeyFunctions.Arcade2);
             human.AddFunction(ContentKeyFunctions.Arcade3);
             human.AddFunction(ContentKeyFunctions.LookUp);
+            human.AddFunction(ContentKeyFunctions.TargetHead);
```

Comment:
will there be a separate "Target" for eyes and mouth? I don't think it's a good idea to combine it into one, especially considering the number of implants in all these parts.

Reply:
At some point yes, along with ears, for now it would be scope creep of the Nth degree to even think of that when most of our prototypes dont even have mouths and such.